packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -205,7 +205,9 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
             } }
             readOnly={ !allowTextInput }
             value={ formattedDate }
-            ref='textField' />
+            ref='textField'
+            role={ allowTextInput ? "combobox" : "menu" }
```

Comment:
Using double quotes instead of single quotes here is upsetting tslint, please change them.